api/envoy/extensions/filters/http/rbac/v3/rbac.proto

```diff
@@ -18,19 +21,37 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#extension: envoy.filters.http.rbac]
 
 // RBAC filter config.
+// [#next-free-field: 6]
 message RBAC {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.http.rbac.v2.RBAC";
 
-  // Specify the RBAC rules to be applied globally.
-  // If absent, no enforcing RBAC policy will be applied.
-  // If present and empty, DENY.
-  config.rbac.v3.RBAC rules = 1;
+  oneof rules_specifier {
+    // Specify the RBAC rules to be applied globally.
+    // If absent, no enforcing RBAC policy will be applied.
+    // If present and empty, DENY.
+    config.rbac.v3.RBAC rules = 1;
```

Comment:
Moving an existing field into a oneof is a breaking change in Go, so we can't do this.  Instead, please document in the comment that only one of these two fields may be set, and add a `[(xds.annotations.field_migrate).oneof_promotion = "rules_specifier"]` annotation to both fields, so that we will move them into the oneof if/when we ever do another major version.

Reply:
Thanks, I will fix that.

Reply:
The fix_format CI does not allow `[(xds.annotations.v3.field_migrate).oneof_promotion = "rules_specifier"]`, how can I fix that?

Reply:
cc @phlax

Reply:
im struggling to find the correct error in CI - not sure of reason - is it to do with proto imports as a guess ?

Reply:
In fact, adding `import` to the proto does not fix the issue. And I believe there is something in tools should be blame.

Like the one "tools/protoxform/utils.py", the module `udpa.annotations.migrate` is imported but `xds.annotations.v3.migrate` is not.

Reply:
quite possible - this tooling is a bit brittle - im happy to review fix/workarounds